ui/pages/create-account/new-account.component.js

```diff
@@ -72,6 +92,7 @@ export default class NewAccountCreateForm extends Component {
               large
               className="new-account-create-form__button"
               onClick={createClick}
+              disabled={Boolean(accountsNames.includes(newAccountName))}
```

Comment:
Maybe to wrap this into a function accountCreateBtnDisabled(), or similar.
Something like the pattern used in `this.renderEditing()`, `this.renderReadonly()`.

Reply:
Fixed in c71199a60e2949308b7659f613fe02a48f46d79d.